decidim-proposals/app/types/decidim/proposals/proposal_type.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # This type represents a ParticipatoryProcess.
```

Comment:
Wrong docs

Reply:
Removing this since types are self-documented (have a description field)